config/webpack.common.js

```diff
@@ -15,7 +15,6 @@ const nodeModules = path.resolve(__dirname, '..', 'node_modules');
 module.exports = {
   entry: {
     index: `${root}/index.tsx`,
-    legacy: `${root}/legacy.tsx`,
```

Comment:
it would have been better if these legacy code deletion were in a separate PR since they are unrelated changes.

Reply:
Agreed.  Removing the original banner caused a lot of errors that eventually lead to be taking out a bunch of legacy files.  Commenting out the code seemed about the same so I decided to just remove the code.